src/main/java/ca/fxco/pistonlib/base/ModCreativeModeTabs.java

```diff
@@ -39,6 +39,8 @@ public class ModCreativeModeTabs {
 
             output.accept(ModItems.OBSIDIAN_SLAB_BLOCK);
             output.accept(ModItems.OBSIDIAN_STAIR_BLOCK);
+
+            output.accept(ModItems.AUTO_CRAFTING_BLOCK);
```

Comment:
> // TODO: Hide item from the creative inventory when disabled, since all crafting blocks placed when disabled won't have a block entity

I think guarding this line by an if check should do the trick?

Reply:
I want to make it possible to toggle config values at any time

Reply:
As I understand it that code is run each time the creative menu is opened, so that shouldn't be a problem. Only issue is if you toggle the config while the creative menu is open I guess.

Reply:
Oh right. I would still like a better solution to really prevent people placing it

Reply:
@SpaceWalkerRS This has been resolved by using Minecrafts FeatureFlags system... by hijacking it for conditionally disabling blocks & items